sda-commons-server-auth/src/test/java/org/sdase/commons/server/opa/health/PolicyExistsHealthCheckTest.java

```diff
@@ -0,0 +1,35 @@
+package org.sdase.commons.server.opa.health;
```

Comment:
There are already tests in [the sda-commons-server-auth-testing module](https://github.com/SDA-SE/sda-dropwizard-commons/blob/fda0809f4394306aa618012bf72d6d0a4f65bd5c/sda-commons-server-auth-testing/src/test/java/org/sdase/commons/server/opa/testing/HealthCheckTest.java).

The main problem with those tests is that they don't check the message and they are missing a test for this specific issue (There is a test that checks if `{"result": null}` is working but not if `{}` also works).

Reply:
You mean OpaResponsesIT, right?

Personally I prefer:
* fast unit tests over slow integration tests
* having tests close to the classes to be tested / didn't expect the tests to be in another module

Reply:
No, I mean HealthCheckTest (which should be named PolicyExistsHealthCheckIT but that's another story). I too think that this test should not be part of the `*-testing` module but this is a wider problem across the whole `*-server-auth` module (that would be solvable, but not in this PR).

Personally I more like the tests that show me what I can expect from the Health Check when a "real" OPA responds.

Reply:
I tried to extend `HealthCheckTest` but can't make the result `null`. The test always gives me a `NullNode`. I.e. it's good to have both tests because `null` results really happen in practice. Is this due to WireMock?

Reply:
I moved the unit test to the same package as the integration test and renamed it as suggested.

Reply:
The default `OpaRule` will only send a `{"result": null}` due to the JSON serialization settings. Something like that might work: `OPA_RULE.mock(wire -> wire.stubFor(post(anyUrl()).willReturn(okJson("{}"))));`

Reply:
It should also be possible to move the IT to the `*server-auth` module. But I guess that you then need a test-dependency to the `*-testing` module. I guess that this wasn't done originally to avoid circular dependencies. But I would expect that `server-auth:test -> server-auth-testing:compile -> server-auth:compile` should work.

Reply:
> OPA_RULE.mock(wire -> wire.stubFor(post(anyUrl()).willReturn(okJson("{}"))));

Yeah, that worked. Thanks! 👍 
I leave the test in "testing" module for now.